Functions/TestResults.ps1

```diff
@@ -33,12 +33,21 @@ function Export-PesterResults {
         [string] $Format
     )
 
-    switch ($Format) {
-        'NUnitXml' {
-            Export-NUnitReport -PesterState $PesterState -Path $Path
+
+    switch ($Format)
```

Comment:
Please use Stroustrup formatting as the rest of the code to avoid unnecessary changes.

Reply:
Ok, should also be fixed. 
![image](https://user-images.githubusercontent.com/6302961/61935477-e5f69180-af8a-11e9-9ac0-6a40cc6b963e.png)